Nakama/INTransport.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+
+namespace Nakama
+{
+    internal interface INTransport
+    {
+        bool Trace { get; set;  }
```

Comment:
Let's remove the setters on `Trace` and `Logger` and initialize the fields in each transport's constructor.